disparate-flies/BangazonSite
# Ay - Seeded Database

Please structure your pull requests this way:

- **Purpose of PR**
Seeded Database



- **Issue number this request impacts** ------
#40


- **Description of what was changed:**
Seeded Database



- **Thorough list of steps to test**
If you already checked this pull request, I've made updates. 
Please delete the migration you have and start over. 

1. git checkout master
2. git fetch --all
3. git checkout ay-startUp
(make sure you're in csharp-bangazon-site-boilerplate)
4. start Bangazon.sln
5. in the Package Manager Console, type 'update-database'
6. check the SQL Server Object Explorer
7. Make sure all of the columns we need are there


if you don't do this you will be
**FIRED**
